### ISSUE TITLE
NTR, pulmo: Intra-alveolar accumulation of phospholipids

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5567]
**Preferred term label:**
Intra-alveolar accumulation of phospholipids

**Synonyms**
Detection of PAS-positive extracellular material in broncho-alveolar lavage

**Definition (free text, please give PubMed ID)**
In the space betweem alveolar macrophages, amorphous PAS-positive material, sometimes as condensed form (oval bodies) are typically found in alveolar proteinosis. 

(https://doi.org/10.1016/j.rmed.2011.08.018) 

**Parent term (use hpo.jax.org/app)**
Abnormal biochemical composition of broncho-alveolar fluid

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 610910, OMIM: 300770, OMIM: 614370 

**Gene association**
CSF2RA (OMIM: 306250), CSF2RB (OMIM: 138981 )

**Your nano-attribution (ORCID)**

**Comment by GrieseAG** [comment]
Comment:  Ther periodic acid Schiff (PAS) staining is a staining method that detects polysaccharidese and mucosubstances, for example: glycogen, glyolipids, glyoproteins and mucins. It is one of the commonly used procedures in the histophathology laboratory and is easy to perfom. It may help in discriminating different forms of interstitial lung disease (ILD), especially pulmonary alveolar proteinosis (PAP).

**Comment by pnrobinson** [comment]
term added

**Comment by pnrobinson** [comment]
@GrieseAG  actually I wonder if this term isn't synonymous with the existing term Alveolar proteinosis
https://hpo.jax.org/app/browse/term/HP:0006517

**Comment by GrieseAG** [comment]
Yes, based on the descpription of Alveolar proteinosis HP:0006517, it could work as a synonymous, at the beginning it seemed like a disease discription for me. Here, the desired term describes the possible result of BAL analysis. But our description could be merged to the already existing term, as well. Because the terms should be used for phenotypic abnormalities, I would prefer our suggested term label: Intra-alveolar accumulation of phospholipids

**Comment by pnrobinson** [comment]
I have merged these terms!
